src/documentation/dsls/sql/create-external-table.md

```diff
@@ -1,8 +1,8 @@
 ---
 layout: section
-title: "Beam SQL: CREATE TABLE Statement"
+title: "Beam SQL: CREATE EXTERNAL TABLE Statement"
 section_menu: section-menu/sdks.html
-permalink: /documentation/dsls/sql/create-table/
+permalink: /documentation/dsls/sql/create-external-table/
 redirect_from: /documentation/dsls/sql/statements/create-table/
```

Comment:
hmm, try changing the redirects to also have the old create-table page. replace the line with this:

redirect_from:
  - /documentation/dsls/sql/statements/create-table/
  - /documentation/dsls/sql/create-table/